docs/docs/ref-01-top-level-api.md

```diff
@@ -19,6 +19,8 @@ function createClass(object specification)
 
 Create a component given a specification. A component implements a `render` method which returns **one single** child. That child may have an arbitrarily deep child structure. One thing that makes components different than standard prototypal classes is that you don't need to call new on them. They are convenience wrappers that construct backing instances (via new) for you.
 
+A React component name must start with an upper-case character. For more information see [HTML Tags vs. React Components] (/react/docs/jsx-in-depth.html#html-tags-vs.-react-components).
```

Comment:
Nit: please remove the space between `]` and `(` (just in case we switch to a different markdown processor that doesn't handle that)